flexget/plugins/input/filesystem.py

```diff
@@ -99,6 +101,11 @@ def prepare_config(self, config):
         config.setdefault('regexp', '.')
         # Sets the default retrieval option to files
         config.setdefault('retrieve', self.retrieval_options)
+        # Sets handing behavior for hidden and dotfiles
+        config.setdefault('skip_hidden_files', self.skip_hidden_files)
+        # Pulls in regexp for ignored files if set
+        if config.get('reject_regexp'):
+            config['reject_regexp'] = translate(config['reject_regexp'])
```

Comment:
Seems wrong to use `translate`. It should already be valid regex. I also prefer `exclude` over `reject`. No action is taken, it just excludes a pattern.